src/vs/workbench/browser/parts/editor/editor.ts

```diff
@@ -88,110 +101,149 @@ export function getEditorPartOptions(configurationService: IConfigurationService
 		options.tabHeight = windowConfig.window.density.editorTabHeight;
 	}
 
-	validateEditorPartOptions(options);
+	return validateEditorPartOptions(options);
+}
+
 
-	return options;
+interface IVerifier<T> {
```

Comment:
I suggest to move the verifiers into a `vs/base` layer so that we can use this in other contexts.